Sources/MapboxMaps/Foundation/Extensions/Core/MapSnapshotOptions.swift

```diff
@@ -11,24 +23,38 @@ extension MapSnapshotOptions {
     ///   - resourceOptions: Resource fetching options to be used by the
     ///         snapshotter. Default uses the access token provided by
     ///         `ResourceOptionsManager.default`
-    public convenience init(size: CGSize,
-                            pixelRatio: CGFloat,
-                            glyphsRasterizationOptions: GlyphsRasterizationOptions? = GlyphsRasterizationOptions(),
-                            resourceOptions: ResourceOptions = ResourceOptionsManager.default.resourceOptions) {
+    public init(size: CGSize,
+                pixelRatio: CGFloat,
+                glyphsRasterizationOptions: GlyphsRasterizationOptions? = GlyphsRasterizationOptions(),
+                resourceOptions: ResourceOptions = ResourceOptionsManager.default.resourceOptions) {
         precondition(pixelRatio > 0)
         precondition(size.width * pixelRatio <= 8192, "Width or scale too great.")
         precondition(size.height * pixelRatio <= 8192, "Height or scale too great.")
 
-        let coreOptions = MapboxCoreMaps.ResourceOptions(resourceOptions)
+        self.size = size
+        self.pixelRatio = pixelRatio
```

Comment:
Do we need to do any magic here with pixel ratio with snapshotter like we had to do for the car play scenario?

Reply:
We should be good here.